tomcat-conf/server.xml

```diff
@@ -68,7 +68,7 @@
     -->
     <Connector port="8080" protocol="HTTP/1.1"
                connectionTimeout="20000"
-               redirectPort="8443" />
```

Comment:
@dbrugman-pega is there any specific reason to remove the redirection port here?

Reply:
Is this similar to [PR ](https://github.com/pegasystems/pega-helm-charts/pull/241)in pega-helm-charts?